src/kubectlUtils.ts

```diff
@@ -111,3 +130,82 @@ export async function switchNamespace(kubectl: Kubectl, namespace: string): Prom
     }
     return true;
 }
+
+/**
+ * Run the specified image in the kubernetes cluster.
+ * 
+ * @param kubectl the kubectl client. 
+ * @param image the docker image.
+ * @param exposedPorts the exposed ports.
+ * @param env the additional environment variables when running the docker container.
+ * @return the deployment name.
+ */
+export async function runAsDeployment(kubectl: Kubectl, image: string, exposedPorts: number[], env: any): Promise<string> {
+    let imageName = image.split(":")[0];
+    let imagePrefix = imageName.substring(0, imageName.lastIndexOf("/")+1);
+    let baseName = imageName.substring(imageName.lastIndexOf("/")+1);
+    const deploymentName = `${baseName}-debug-${Date.now()}`;

```

Comment:
This doesn't seem to be in the debug directory, so it shouldn't be hardwired to use debug deployment names.